From 5a6449c3b3d9bbc2b1bc15a5d5b40d7228f3a406 Mon Sep 17 00:00:00 2001
From: Dominik Haumann <dhaumann@kde.org>
Date: Wed, 30 Jan 2019 17:16:32 +0100
Subject: Fix bug 403422: Allow changing the marker size again

This bug was introduced with https://phabricator.kde.org/D17661
with commit c287d3858fd780f5e139d05e8bb7d4558ccc71b8

FIXED-IN: 5.55
BUG: 403422
---
 src/dialogs/textareaappearanceconfigwidget.ui | 27 +++++++++++++++++++++------
 1 file changed, 21 insertions(+), 6 deletions(-)

diff --git a/src/dialogs/textareaappearanceconfigwidget.ui b/src/dialogs/textareaappearanceconfigwidget.ui
index 02a0d54..cc44a39 100644
--- a/src/dialogs/textareaappearanceconfigwidget.ui
+++ b/src/dialogs/textareaappearanceconfigwidget.ui
@@ -185,10 +185,10 @@
             <string>Size of the visible highlight marker.</string>
            </property>
            <property name="minimum">
-            <number>-1</number>
+            <number>1</number>
            </property>
            <property name="maximum">
-            <number>-1</number>
+            <number>5</number>
            </property>
            <property name="singleStep">
             <number>1</number>
@@ -301,6 +301,21 @@ beginning of a file.</string>
    <header>kcombobox.h</header>
   </customwidget>
  </customwidgets>
+ <tabstops>
+  <tabstop>gbWordWrap</tabstop>
+  <tabstop>chkDynWrapAtStaticMarker</tabstop>
+  <tabstop>cmbDynamicWordWrapIndicator</tabstop>
+  <tabstop>sbDynamicWordWrapDepth</tabstop>
+  <tabstop>chkShowTabs</tabstop>
+  <tabstop>chkShowSpaces</tabstop>
+  <tabstop>sliSetMarkerSize</tabstop>
+  <tabstop>chkShowIndentationLines</tabstop>
+  <tabstop>chkShowWholeBracketExpression</tabstop>
+  <tabstop>chkAnimateBracketMatching</tabstop>
+  <tabstop>chkFoldFirstLine</tabstop>
+  <tabstop>chkShowWordCount</tabstop>
+  <tabstop>chkShowLineCount</tabstop>
+ </tabstops>
  <resources/>
  <connections>
   <connection>
@@ -314,8 +329,8 @@ beginning of a file.</string>
      <y>7</y>
     </hint>
     <hint type="destinationlabel">
-     <x>340</x>
-     <y>44</y>
+     <x>559</x>
+     <y>117</y>
     </hint>
    </hints>
   </connection>
@@ -330,8 +345,8 @@ beginning of a file.</string>
      <y>6</y>
     </hint>
     <hint type="destinationlabel">
-     <x>385</x>
-     <y>72</y>
+     <x>559</x>
+     <y>162</y>
     </hint>
    </hints>
   </connection>
-- 
cgit v1.1

